x-pack/plugins/infra/public/types.ts

```diff
@@ -90,6 +91,7 @@ export interface InfraClientSetupDeps {
   lens: LensPublicStart;
   fieldFormats: FieldFormatsSetup;
   licensing: LicensingPluginSetup;
+  cloud?: { isCloudEnabled: boolean; cloudId?: string; isServerlessEnabled: boolean };
```

Comment:
Can we use `CloudSetup` here to keep the same pattern or there is some issue with it?

Reply:
Done ✅